### ISSUE TITLE
Create script to automate updates of effective_tld_names

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#14941]
We’re currently never updating our `effective_tld_names.dat` [file](https://github.com/mozilla-mobile/firefox-ios/blob/main/Shared/effective_tld_names.dat). We should have something in place to automate that update for us, similar to what Gecko has. 

The script that does that syncing for Gecko is [https://searchfox.org/mozilla-central/source/taskcluster/docker/periodic-updates/scripts/periodic_file_updates.sh](https://searchfox.org/mozilla-central/source/taskcluster/docker/periodic-updates/scripts/periodic_file_updates.sh|smart-link). It runs via cron twice a week. We maybe don’t need to update that often in our case, once or twice a cycle would probably be fine.

Basically what the script is doing is fetching [https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat](https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat) and diffing the current in-tree version.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-6697)

**Comment by data-sync-user** [comment]
➤ Isabel Rios Escobar commented:

Looking into this issue as it may be possible to automate via github actions

**Comment by data-sync-user** [comment]
➤ Isabel Rios Escobar commented:

Completed as part of MTE-1141 ( https://mozilla-hub.atlassian.net/browse/MTE-1141 )
